datastore/datastore_test.go

```diff
@@ -68,12 +68,12 @@ func TestKVObjectFlatKey(t *testing.T) {
 func TestAtomicKVObjectFlatKey(t *testing.T) {
 	store := NewTestDataStore()
 	expected := dummyKVObject("1111", true)
-	assert.False(t, expected.Exists())
+	assert.Check(t, !expected.Exists())
 	err := store.PutObjectAtomic(expected)
 	if err != nil {
```

Comment:
For someone that has some time to spend; these can be changed to use `assert.NilError(t, err)`